src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -27,6 +27,7 @@ void regclass_Core(py::module m) {
 
     cls.def(py::init<const std::string&>(), py::arg("xml_config_file") = "");
 
+    // todo: remove after Accuracy Checker migration to set/get_property API
```

Comment:
is it possible to deprecate method in pybind?

Reply:
yep, done